apps/ctrlpad/main.ts

```diff
@@ -193,8 +193,9 @@
 			{
 				text: "DnD",
 				cb: tap => {
-					let on;
-					if(on = !!origBuzz){
+					let on = false;
+					if(origBuzz){
```

Comment:
Here my intention was to indicate via code that `on` is derived from `origBuzz`, we can keep this and satisfy the linter:
```suggestion
					if(((on = !!origBuzz)){
```

Reply:
I don't like this solution since you're not honoring the idea with the lint rules, which is that it is hard to read and understand that this is both an assignment and a boolean check. But I don't want to argue any more about it, so this is fine.